cmd/podman/cleanup.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/projectatomic/libpod/cmd/podman/libpodruntime"
+	"github.com/projectatomic/libpod/libpod"
+	"github.com/urfave/cli"
+)
+
+var (
+	cleanupFlags = []cli.Flag{
+		cli.BoolFlag{
+			Name:  "all, a",
+			Usage: "Remove all containers",
```

Comment:
Remove all?

Also, does this flag even make sense? If we ever end up in a situation where we need to manually clean up every container, something has gone seriously wrong.

Reply:
@mheon that's a good point, it really only cleans all exited containers and probably should say that if we're keeping the --all param.

Reply:
Cut and paste, fixed.